test/controllers/manageDetails/UpdateOrganisationDetailsSpec.scala

```diff
@@ -247,10 +247,14 @@ class UpdateOrganisationDetailsSpec extends VoaPropertyLinkingSpec {
       mockGroups,
       mockAddresses,
       mockManageDetails,
-      updateBusinessAddressView = new views.html.details.updateBusinessAddress(mainLayout, GovukButton, GovukInput),
-      updateBusinessNameView = new views.html.details.updateBusinessName(mainLayout, GovukButton, GovukInput),
-      updateBusinessPhoneView = new views.html.details.updateBusinessPhone(mainLayout, GovukButton, GovukInput),
-      updateBusinessEmailView = new views.html.details.updateBusinessEmail(mainLayout, GovukButton, GovukInput)
+      updateBusinessAddressView =
+        new views.html.details.updateBusinessAddress(mainLayout, GovukButton, GovukInput, FormWithCSRF),
+      updateBusinessNameView =
+        new views.html.details.updateBusinessName(mainLayout, GovukButton, GovukInput, FormWithCSRF),
+      updateBusinessPhoneView =
+        new views.html.details.updateBusinessPhone(mainLayout, GovukButton, GovukInput, FormWithCSRF),
+      updateBusinessEmailView =
+        new views.html.details.updateBusinessEmail(mainLayout, GovukButton, GovukInput, FormWithCSRF)
```

Comment:
how do you fancy moving these into FakeViews?

Reply:
+1